shared/src/commonMain/kotlin/org/hyperskill/app/track_selection/details/presentation/TrackSelectionDetailsActionDispatcher.kt

```diff
@@ -26,20 +27,9 @@ class TrackSelectionDetailsActionDispatcher(
 ) : CoroutineActionDispatcher<Action, Message>(config.createConfig()) {
     override suspend fun doSuspendableAction(action: Action) {
         when (action) {
-            is TrackSelectionDetailsFeature.InternalAction.FetchAdditionalInfo -> {
-                val transaction =
-                    HyperskillSentryTransactionBuilder.buildTrackSelectionDetailsScreenRemoteDataLoading()
-                sentryInteractor.startTransaction(transaction)
-                val message = fetchProvidersAndSubscription(action)
-                    .getOrElse {
-                        sentryInteractor.finishTransaction(transaction, it)
-                        onNewMessage(TrackSelectionDetailsFeature.TrackSelectionResult.Error)
```

Comment:
Also here we return incorrect error message `TrackSelectionDetailsFeature.TrackSelectionResult.Error`, must be `TrackSelectionDetailsFeature.FetchAdditionalInfoResult.Error`.

So in case of remote data loading user will see infinite loading state.